module/zfs/vdev.c

```diff
@@ -4438,23 +4438,27 @@ vdev_get_stats_ex_impl(vdev_t *vd, vdev_stat_t *vs, vdev_stat_ex_t *vsx)
 }
 
 /*
- * Get mirror slog alloc
+ * Get mirror slog alloc and space
  */
-static uint64_t
-getMirrorSlogUsedSize(vdev_t* vd)
+static void
+getMirrorSlogSizeDetails(vdev_t *vd, unsigned __int64 *mirror_slog_alloc_size,
+    unsigned __int64 *mirror_slog_space)
 {
     if (!vd->vdev_ops->vdev_op_leaf) {
 	for (int c = 0; c < vd->vdev_children; c++) {
 	    vdev_t* cvd = vd->vdev_child[c];
 	    vdev_stat_t* cvs = &cvd->vdev_stat;
 	    vdev_stat_ex_t* cvsx = &cvd->vdev_stat_ex;
 
-	    getMirrorSlogUsedSize(cvd);
-	    if (cvd->vdev_islog)
-		return cvs->vs_alloc;
+	    getMirrorSlogSizeDetails(cvd, &mirror_slog_alloc_size,
+		&mirror_slog_space);
```

Comment:
getting below compiler warnings. can you pls check?

C:\mergezfsdestroy\openzfs\module\zfs\vdev.c(4453,36): warning : incompatible pointer types passing 'unsigned long long **' to parameter of type 'unsigned long long *'; remove & [-Wincompatible-pointer-types]
              getMirrorSlogSizeDetails(cvd, &mirror_slog_alloc_size,
                                            ^~~~~~~~~~~~~~~~~~~~~~~
  ..\..\..\module\zfs\vdev.c(4444,56): note: passing argument to parameter 'mirror_slog_alloc_size' here
  getMirrorSlogSizeDetails(vdev_t *vd, unsigned __int64 *mirror_slog_alloc_size,
                                                         ^
C:\mergezfsdestroy\openzfs\module\zfs\vdev.c(4454,3): warning : incompatible pointer types passing 'unsigned long long **' to parameter of type 'unsigned long long *'; remove & [-Wincompatible-pointer-types]
                  &mirror_slog_space);
                  ^~~~~~~~~~~~~~~~~~
  ..\..\..\module\zfs\vdev.c(4445,23): note: passing argument to parameter 'mirror_slog_space' here
      unsigned __int64 *mirror_slog_space)
                        ^
C:\mergezfsdestroy\openzfs\module\zfs\vdev.c(4451,22): warning : unused variable 'cvsx' [-Wunused-variable]
              vdev_stat_ex_t* cvsx = &cvd->vdev_stat_ex;
                              ^
  3 warnings generated.

Reply:
Resolved!